internal/exe/exe.go

```diff
@@ -0,0 +1,163 @@
+package exe
+
+import (
+	"context"
+	"errors"
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/livebud/bud/internal/once"
+)
```

Comment:
This package simplifies the previous exe package to consolidate how subprocesses are managed in bud. Instead of wrapping exec.Command, it can turn an exec.Cmd into a Process.

Reply:
Additionally, it can allow you to configure commands earlier with shared env / stdio / files etc. that can be spawned at a later time.